pvlib/iotools/nasa_power.py

```diff
@@ -0,0 +1,141 @@
+"""Functions for reading and retrieving data from NASA POWER."""
+
+import pandas as pd
+import requests
+
+URL = 'https://power.larc.nasa.gov/api/temporal/hourly/point'
+
+VARIABLE_MAP = {
+    'ALLSKY_SFC_SW_DWN': 'ghi',
+    'ALLSKY_SFC_SW_DIFF': 'dhi',
+    'ALLSKY_SFC_SW_DNI': 'dni',
+    'CLRSKY_SFC_SW_DWN': 'ghi_clear',
+    'T2M': 'temp_air_2m',
```

Comment:
```suggestion
    'T2M': 'temp_air',
```
2 m air temperature is relatively standard, so this should be mapped to ``temp_air``. If users for some reason want a 10 m air temperature, then that's still possible with the T10M parameter.